yolapy/models/user.py

```diff
@@ -0,0 +1,54 @@
+from yoconfig import get_config
+from yolapy.services import Yola
+
+
+class User(object):
+
+    """Yola User."""
+
+    def __init__(self, active=False, deleted=None, email=None, id=None,
+                 name=None, partner_id=None, preferences=None,
+                 signup_date=None, surname=None):
+        """Sets all user attributes and inits a Yola client."""
+        self._partner_id = partner_id
+        self.active = active
+        self.deleted = deleted
+        self.email = email
```

Comment:
At some point, we have to abstract this really nasty bit of the Yola WL API. I'm not sure where the right place is.